examples/config_files/settings/settings_user.rst

```diff
@@ -1,6 +1,250 @@
+    ...
+    Profile host:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=gcc
+    compiler.libcxx=libstdc++11
+    compiler.version=13.0-rc
+    os=Macos
+
+    Profile build:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+    ...
+    -------- Installing (downloading, building) binaries... --------
+    pkg/1.0: Copying sources to build folder
+    pkg/1.0: Building your package in /Users/franchuti/.conan2/p/t/pkg918904bbca9dc/b
+    pkg/1.0: Aggregating env generators
+    pkg/1.0: Package '44a4588d3fe63ccc6e7480565d35be38d405718e' built
+    pkg/1.0: Build folder /Users/franchuti/.conan2/p/t/pkg918904bbca9dc/b
+    pkg/1.0: Generated conaninfo.txt
+    pkg/1.0: Generating the package
+    pkg/1.0: Temporary package folder /Users/franchuti/.conan2/p/t/pkg918904bbca9dc/p
+    pkg/1.0 package(): WARN: No files in this package!
+    pkg/1.0: Package '44a4588d3fe63ccc6e7480565d35be38d405718e' created
+    pkg/1.0: Created package revision d913ec060e71cc56b10768afb9620094
+    pkg/1.0: Full package reference: pkg/1.0#637fc1c7080faaa7e2cdccde1bcde118:44a4588d3fe63ccc6e7480565d35be38d405718e#d913ec060e71cc56b10768afb9620094
+    pkg/1.0: Package folder /Users/franchuti/.conan2/p/pkg789b624c93fc0/p
+
+.. code-block:: bash
+    :caption: **Using the new OS and the new architecture**
+    :emphasize-lines: 5,11,33
+
+    $ conan create . -s os=webOS -s arch=cortexa15t2hf
+    ...
+    Profile host:
+    [settings]
+    arch=cortexa15t2hf
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=webOS
+
+    Profile build:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+    ...
+    -------- Installing (downloading, building) binaries... --------
+    pkg/1.0: Copying sources to build folder
+    pkg/1.0: Building your package in /Users/franchuti/.conan2/p/t/pkgde9b63a6bed0a/b
+    pkg/1.0: Aggregating env generators
+    pkg/1.0: Package '19cf3cb5842b18dc78e5b0c574c1e71e7b0e17fc' built
+    pkg/1.0: Build folder /Users/franchuti/.conan2/p/t/pkgde9b63a6bed0a/b
+    pkg/1.0: Generated conaninfo.txt
+    pkg/1.0: Generating the package
+    pkg/1.0: Temporary package folder /Users/franchuti/.conan2/p/t/pkgde9b63a6bed0a/p
+    pkg/1.0 package(): WARN: No files in this package!
+    pkg/1.0: Package '19cf3cb5842b18dc78e5b0c574c1e71e7b0e17fc' created
+    pkg/1.0: Created package revision f5739d5a25b3757254dead01b30d3af0
+    pkg/1.0: Full package reference: pkg/1.0#637fc1c7080faaa7e2cdccde1bcde118:19cf3cb5842b18dc78e5b0c574c1e71e7b0e17fc#f5739d5a25b3757254dead01b30d3af0
+    pkg/1.0: Package folder /Users/franchuti/.conan2/p/pkgd154182aac59e/p
+
+
```

Comment:
Finish with a ``conan list pkg/1.0:*`` that shows the different built binaries with the custom config.

Reply:
Now much better! 👍